shared/src/proxies/messages/replyToMessageProxy.ts

```diff
@@ -1,24 +1,11 @@
 import { post } from '../requests';
 
-/**
- * replyToMessageInteractor
- *
- * @param {object} applicationContext the application context
- * @param {object} providers the providers object
- * @param {array} providers.attachments array attachments on the message
- * @param {string} providers.docketNumber the docket number of the case
- * @param {string} providers.message the message text
- * @param {string} providers.parentMessageId the id of the parent message for the thread
- * @param {string} providers.subject the message subject
- * @param {string} providers.toSection the section of the user receiving the message
- * @param {string} providers.toUserId the user id of the user receiving the message
- * @returns {Promise<*>} the promise of the api call
- */
 export const replyToMessageInteractor = (
   applicationContext,
   {
     attachments,
     docketNumber,
+    draftAttachments,
     message,
     parentMessageId,
     subject,
```

Comment:
if we're removing jsdocs, we need to type the parameters of the function

Reply:
Agreed. Fixed: https://github.com/ustaxcourt/ef-cms/commit/693e0e6382ec37e511ae30a3d69522d1be3c48cc